src/process.ts

```diff
@@ -0,0 +1,63 @@
+function zip(firstArray: string[], secondArray: string[]) {
+  let str = "";
+  for (let i = 0; i < firstArray.length; i++) {
+    if (secondArray[i]) {
+      str += firstArray[i] + secondArray[i];
+    } else {
+      str += firstArray[i];
+    }
+  }
+
+
+
```

Comment:
Doesn't need to be this PR, but we should probably add the `npm run prettier` and `npm run prettier:fix` scripts to this package.  
When we do, having so many returns in a row will violate our pretty rules.

Reply:
Great point! I've added prettier and used it. Spaces should be gone now.